Library/ENV/4.3/cc

```diff
@@ -276,6 +281,11 @@ class Cmd
     %W[#{sysroot}/usr/lib /usr/local/lib]
   end
 
+  def keg_path
+    return nil if formula.nil?
+    "#{cellar}/#{formula}/#{formula_version}"
```

Comment:
I meant for getting prefix from here, but I guess not. One potential issue with how it is here is that it will silently use empty strings and silently coerce nil to an empty string, but neither is probably a significant risk.